### DISCUSSION TITLE
Fixtures data refresh schedule

### DISCUSSION THREAD

**Discussion by Yashar665** [georgedouzas/sports-betting#70] [General]
Hi, thank you for the brilliant repo! I noticed that the fixtures data is currently empty. Could you please let me know the schedule for refreshing it?

**Comment by georgedouzas** [comment]
Hi @Yashar665,

Usually is updated 1-2 times per week. The data source I use is https://www.football-data.co.uk

A Prefect scheduler downloads the data every 3 hours and updates the `data` branch.
